workshop/content/docs/api-deep-dive/styles.md

```diff
@@ -0,0 +1,455 @@
+---
+title: OGC API - Styles
+---
+
+# OGC API - Styles
+
+!!! abstract "Audience"
+    Students that are familiar with web services and APIs, and want to have an overview of OGC API - Styles standard
+
+!!! abstract "Learning Objectives"
+    At the completion of the module students will be able to:
+
+    - Explain what the OGC API - Styles standard is
+    - Describe what can be done with OGC API - Styles implementations
+    - Understand the main resources offered by OGC API - Styles implementations
+    - Understand how to retrieve a description of the capabilities of an OGC API - Styles implementation
+    - Understand how to issue requests to an implementation of OGC API - Styles
+    - Be able to find an OGC API - Styles endpoint and use it through a client
+
+## Introduction
+
+[OGC API - Styles](https://ogcapi.ogc.org/styles) is a standard that describes an API that enables map servers, clients as well as visual style editors, to manage and fetch styles. The styles consist of symbolizing instructions that can be applied by a rendering engine on features and/or coverages. The API implements the conceptual model for style encodings and style metadata.
+
+!!! note
+    This tutorial module is not intended to be a replacement to the actual
+    **OGC API - Styles - Part 1: Core** standard. The tutorial intentionally
+    focuses on a subset of capabilities in order to get the student started
+    with using the standard. Please refer to the [**OGC API - Styles - Part 1:
+    Core** standard](https://docs.ogc.org/DRAFTS/20-009.html) for additional detail.
+
+### Background
+
+> History
+
+The need for users and software to be able to control the visual portrayal of geospatial data was already present in the first generation of OGC Web Services. A profile of the [Web Map Service (WMS)](https://www.ogc.org/standard/wms/) Standard was published in 2007, for the definition of a [Styled Layer Descriptor (SLD)](https://www.ogc.org/standard/sld/); this profile defined an encoding that extends the WMS standard to allow user-defined symbolization and coloring of geographic feature and coverage data. 
```

Comment:
In 2001, WMS 1.1.0 introduced enhanced support for styles using the Styled Layer Descriptor (SLD) standard.